docs/openapi_v1.0.0-alpha.yaml

```
@@ -1204,7 +1263,7 @@ components:
         format: string
     Uuid:
       name: 'uuid'
-      in: query
+      in: path
       description: |
         A valid booking reservation UUID that matches the `uuid` sent during the initial `POST /reservations`
```

Need to update this description to match the new endpoint structure.

```suggestion
         A valid booking reservation UUID that matches the `uuid` sent during the initial `POST /suppliers/{supplierId}/bookings`
```